build/stf-run-ci/tasks/clone_repos.yml

```diff
@@ -58,3 +58,20 @@
         repo: https://github.com/infrawatch/prometheus-webhook-snmp
         dest: working/prometheus-webhook-snmp
         version: master
+
+# Branches for Loki Operator don't work the same as with other repositories.
+# We don't have write access to the upstream repository to create our own
+# branches there.
+- name: Get Loki Operator
+  block:
+    - name: Try cloning same-named branch or override branch
+      git:
+        repo: https://github.com/viaq/loki-operator
```

Comment:
I'm not suggesting you change this now, but we could possibly make this a default parameter that could be overridden, and then if we forked it for the purposes of development, then at least you could consume that forked version while doing the development.

This is basically the case for all the repositories though, not just this one (although of course we as developers do have access to custom branches on the repo). Something we can deal with if this becomes a use case in the future though.

Reply:
I might have some time later this sprint, so I may take a look at this.

Reply:
The time came earlier than I thought :D